src/client/containers/Channel/Channel.js

```diff
@@ -51,20 +59,47 @@ export default function Channel() {
       </>
     );
   }
-  if (messages.length === 0) {
+  if (channel?.title === null) {
```

Comment:
This could be simpler: extract properties for the head nav: `channelName`, `channelId` and `imgUrl` before and the you will have a single return statement at the end of the file.